README.md

```diff
@@ -43,7 +44,7 @@ _Note: You can also build your own secret manager by subclassing the `CustomSecr
 
 ### Retrieving Secrets
 
-```python
+\`\`\`python
```

Comment:
Here and elsewhere in this file, I don't think you need to escape the triple backtick fencing.

Reply:
@ellipsis-dev Can you try to implement the requested change?